src/app-layout/utils/use-pointer-events.ts

```diff
@@ -12,21 +12,13 @@ import { SizeControlProps } from './interfaces';
 
 import styles from '../resize/styles.css.js';
 
-export const usePointerEvents = ({
-  position,
-  panelRef,
-  handleRef,
-  onResize,
-  hasTransitions = false,
-}: SizeControlProps) => {
+export const usePointerEvents = ({ position, panelRef, handleRef, onResize }: SizeControlProps) => {
   const onDocumentPointerMove = useCallback(
     (event: PointerEvent) => {
       if (!panelRef || !panelRef.current || !handleRef || !handleRef.current) {
         return;
       }
 
-      panelRef.current.classList.remove(styles['with-motion']);
```

Comment:
We can remove this JS handling and do it with CSS only: `.with-motion-vertical:not(.resize-active)`